spec/features/cancel_spec.rb

```diff
@@ -1,80 +1,95 @@
-describe "Canceling a request" do
-  let!(:client_data) { FactoryGirl.create(:ncr_work_order, :with_approvers) }
-  let!(:proposal) { client_data.proposal }
-
-  it "shows a cancel link for the requester" do
+describe 'Canceling a request' do
+  it 'shows a cancel link for the requester' do
+    proposal = FactoryGirl.create(:proposal)
     login_as(proposal.requester)
+
     visit proposal_path(proposal)
-    expect(page).to have_content("Cancel my request")
+
+    expect(page).to have_content('Cancel my request')
   end
 
-  it "does not show a cancel link for non-requesters" do
+  it 'does not show a cancel link for non-requesters' do
+    proposal = FactoryGirl.create(:proposal, :with_approver)
     login_as(proposal.approvers.first)
+
     visit proposal_path(proposal)
-    expect(page).to_not have_content("Cancel my request")
+
+    expect(page).to_not have_content('Cancel my request')
   end
 
-  it "prompts the requester for a reason" do
+  it 'prompts the requester for a reason' do
+    proposal = FactoryGirl.create(:proposal)
     login_as(proposal.requester)
+
     visit proposal_path(proposal)
     click_on('Cancel my request')
+
     expect(current_path).to eq("/proposals/#{proposal.id}/cancel_form")
   end
 
   context 'email' do
-    # TODO this shouldn't actually be needed
-    around(:each) { ActionMailer::Base.deliveries.clear }
-
-    it "send emails when cancellation is complete" do
+    it 'send emails when cancellation is complete' do
+      ActionMailer::Base.deliveries.clear
+      proposal = FactoryGirl.create(:proposal)
       expect(deliveries.length).to eq(0)
 
       login_as(proposal.requester)
       visit proposal_path(proposal)
       click_on('Cancel my request')
-      fill_in "reason_input", with: "This is a good reason for the cancellation."
+      fill_in 'reason_input', with: 'This is a good reason for the cancellation.'
       click_on('Yes, cancel this request')
 
-      expect(deliveries.length).to eq(2)
+      expect(deliveries.length).to eq(1)
```

Comment:
it passes as-is, but the previous expectation was to have 2 emails. So...this might be something to ask @phirefly , who added this test in https://github.com/18F/C2/commit/60cbf244ed6fa2e2cdc0e1fe429c145d9c14d48c